src/server/models/index.js

```diff
@@ -383,6 +424,18 @@ async function getItems(req) {
     setAsAvailable: true,
   });
 
+  const agregationsMultiFilter = [];
+  Object.entries(searchParamsMultiFilters).forEach((searchParamsMultiFilter, currentIndex) => {
```

Comment:
Sofern nur der Key benötigt wird, kann denke ich auch `Object.keys()` genutzt werden.

Anstatt `forEach()` könnte man hier auch `reduce()` nuzten ;)

```javascript
const agregationsMultiFilter = Object.keys(searchParamsMultiFilters).reduce((acc, filterKey, currentIndex) => {
    const currentAggregation = aggregateESFilterBuckets({
      aggregations: result.body.responses[currentIndex + 2].aggregations,
      setAsAvailable: true,
    });

    // TODO: Das geht bestimmt auch eleganter
    acc[filterKey] = currentAggregation[filterKey];
}, {});
```

Soll `agregationsMultiFilter` zudem eigentlich ein Array oder ein Objekt sein?

Reply:
Das ist nach der anschließenden Zuweisung eigentlich ein Objekt oder?

Reply:
Es wird in Zeile 427 als Array initialisiert.

In meinem Code-Beispiel wäre es ein Objekt. In deinem Code wäre es ein Array, dem Eigenschaften zugewiesen werden. Hängt auch ein wenig davon ab, wie es weitergenutzt wird.

Reply:
Ich habe jetzt ein Objekt daraus gemacht. Da ich anschließend `Object.entries` darauf anwende.

Reply:
Ok